docs/file-events.md

```diff
@@ -0,0 +1,107 @@
+- **Content Document Link Event** - Use this event if yu need to trigger automation when a Content Document Link is Created. A Content Document Link connects a Salesforce record (like an Account or a Case) to a Content Document. This could be used to take an action on a related object when a file is uploaded to it or to Adjust/Default a File Tag based on the object it is uploaded to.
+- **Content Version Event** - Use this event if you want to trigger automation when a Content Version is Updated. This could be used to trigger automation based on a File Tag or to build or update an Analog Object to use Salesforce reporting on your tags.
+
+Once you have chosen your Platform Event you can build automation like in a Record Triggered Flow. The next section will describe the fields on the three Platform Events in depth.
+
+## File Event Platform Event Glossary
+
+### Content Document Event
+
+Action - This field will tell you what context the platform event was created in. It can be one of the following values:
+
+- afterInsert
+- afterUpdate 
+- afterDelete
+- afterUndelete
+
+Content Document Id - This is the Content Document Id that created the Platform Event.
+
+### Content Document Link Event
+
+Action - This field will tell you what context the platform event was created in. It can be one of the following values:
+
+- afterInsert
+- afterUpdate 
+- afterDelete
+- afterUndelete
+
+Content Document Id - This is the Content Document Id that created the Platform Event. 
+
+Content Document Link Id - This is the Id of the Content Document Link. This is a junction object that connects a Content Document and any other File enabled sObject.
+
+Linked Entity Id - This is the Id for the sObject that is connected to the Content Document via the Content Document Link. This is a polymorphic lookup so it can be any object that allows files. In a Flow that uses this Id you will need to ascertain what the Object Name is before doing Get Elements to prevent errors.
+
+### Content Version Event
+
+Action - This field will tell you what context the platform event was created in. It can be one of the following values:
+
+- afterInsert
+- afterUpdate 
+- afterDelete
+- afterUndelete
+
+Content Document Id - This is the Content Document Id that created the Platform Event.
+
+Content Version Id - This is the Id of the Content Version. The Content Version is where the Tagging Data is stored in Salesforce. Use this object to view and update tags on the Files.
+
+## Considerations When Using File Events
+
+- If you are using Bulk Upload in FileViewer you will not be able to use the Content Version Event and will need to turn fully deactivate the Content Version Apex Trigger Setting. If you do not, it could result in the following error:

```

Comment:
```suggestion
- If you are using Bulk Upload feature within FileViewer (below version 0.26) you will not be able to use the Content Version Event and will need to turn fully deactivate the Content Version Apex Trigger Setting. If you do not, it could result in the following error:

```